core/src/main/scala/kafka/admin/AdminUtils.scala

```diff
@@ -564,6 +564,11 @@ object AdminUtils extends Logging {
     topics.map(topic => fetchTopicMetadataFromZk(topic, zkUtils, cachedBrokerInfo))
   }
 
+  def fetchTopicMetadataFromZkWithProtocol(topics: Set[String], zkUtils: ZkUtils, protocol: SecurityProtocol = SecurityProtocol.PLAINTEXT): Set[MetadataResponse.TopicMetadata] = {
```

Comment:
I suggest naming this `fetchTopicMetadataFromZK`, remove the default argument from `protocol` and then call this method from the other `fetchTopicMetadataFromZk`. What do you think?